FHICORC/Views/NativeLoadingPage.xaml

```diff
@@ -1,11 +1,43 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
-             x:Class="FHICORC.Views.NativeLoadingPage">
```

Comment:
Please add a ref to the viewmodel datatype. Properly needs to look something like this:
xmlns:viewmodels="clr-namespace:FHICORC.ViewModels.NativeLoadingViewModels" x:DataType="viewmodels:NativeLoadingViewModel"